qlkube/src/index.js

```diff
@@ -50,6 +50,28 @@ async function main() {
     schema,
     playground: true,
     plugins: [graphqlQueryRegistry],
+    formatError: error => {
+      const msgs = [...error.message.match(/(\d+)\s*\-(.*)$/)];
```

Comment:
Trying a wrong query in order to check the result obtained I get:
`"message": "error.message.match is not a function or its return value is not iterable"` instead of `"message": "Context creation failed: Token Error! Token absent!"`.
The query used is:
```
{
  itPolitoCrownlabsV1alpha1Workspace(name: "prova"){
    kind
  }
}
```

Reply:
Thanks! I guess wrapping everything in the try/catch can be more robust